cmd/init.go

```diff
@@ -0,0 +1,160 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"path"
+	"text/template"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"go.k6.io/k6/cmd/state"
+	"go.k6.io/k6/lib/fsext"
+)
+
+const defaultNewScriptName = "script.js"
+
+//nolint:gochecknoglobals
+var (
+	errFileExists            = errors.New("file already exists")
+	defaultNewScriptTemplate = template.Must(template.New("init").Parse(`import http from 'k6/http';
+import { sleep } from 'k6';
+
+export const options = {
+  // A number specifying the number of VUs to run concurrently.
+  vus: 10,
+  // A string specifying the total duration of the test run.
+  duration: '30s',
+
+  // The following section contains configuration options for execution of this
+  // test script in Grafana Cloud.
+  //
+  // See https://grafana.com/docs/grafana-cloud/k6/get-started/run-cloud-tests-from-the-cli/
+  // to learn about authoring and running k6 test scripts in Grafana Cloud.
+  //
+  // cloud: {
+  //   // The ID of the project to which the test is assigned in the k6 Cloud UI.
+  //   // By default tests are executed in default project.
+  //   projectID: "",
+  //   // The name of the test in the k6 Cloud UI.
+  //   // Test runs with the same name will be grouped.
+  //   name: "{{ .ScriptName }}"
+  // },
+
+  // Uncomment this section to enable the use of Browser API in your tests.
+  //
+  // See https://k6.io/docs/using-k6-browser/running-browser-tests/ to learn more
+  // about using Browser API in your test scripts.
+  //
+  // scenarios: {
+  //   // The scenario name appears in the result summary, tags, and so on.
+  //   SCENARIO NAME: {
```

Comment:
```suggestion
  //   "browser example scenario": {
```
I would prefer if we provide only _ready to use_ code.

Reply:
I've been thinking about this too and ended up with making this JSON incorrect intentionally. I want to draw user attention to this section and encourage them to read it through if they decide to opt-in for the Browser API. My concern is that if I use something like `ui` or `browser` there, this might give an impression that this is a mandatory name.

Another option I had is to have `exec` there and add another commented block with a browser-based test. WDYT?

Reply:
> I think you may expand this comment on the previous line making clear that the name is configurable and it is a good practice to pick a meaningful name for the customer's business context.

That's a way to solve it, but I'm quite pessimistic about how often people read comments unless things go wrong 😅

Reply:
Let's see if the other reviewers have some other suggestions otherwise, I'm fine with your current proposal.

Reply:
I tend to agree with the point raised by @codebien, especially considering that this comment section starts with the statement "uncomment this section to enable". Technically, and to be nitpicky (for once), if I uncomment this section, the script won't work (right?).

> I'm quite pessimistic about how often people read comments

I see where you're coming from, but my own personal experience is that I do really appreciate comments in an automatically generated file to guide me forward. I don't immediately read them, but I do as soon as I'm blocked. My assumption is that the init file is there to get started quickly, and I anticipate that as users introduce complexity in their script, they might rely on comments more and more if they're present 😄

All of this to say I'm rather for having a valid name and expanding on the comment a bit more too 👍🏻